README.md

```diff
@@ -547,6 +548,61 @@ instance.appendAppSubversion(8888);
 instance.connect();
 ```
 
+## <a id="getattributioninfo"></a>getAttributionInfo method
+
+Description on how to use getAttributionInfo method.
```

Comment:
Change it from description to explain the customer in a brief sentence what it is about and how it is helpful.